src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+// PeoplePage.js
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { PeopleList } from './PeopleList';
+
+type Person = {
+  name: string;
+  sex: string;
+  born: number;
+  died: number;
+  mother: string;
+  father: string;
+  slug: string;
+};
+
+export const PeoplePage = () => {
+  const { personHref } = useParams(); // Получаем personHref из параметров URL
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(fetchedPeople => {
+        setPeople(fetchedPeople);
+        setError(null);
+      })
+      .catch(() => setError('Something went wrong'))
```

Comment:
The error message 'Something went wrong' is quite generic. Consider providing more detailed error information if available, or logging the error to the console for debugging purposes.